wrf/utils.py

```diff
@@ -503,9 +518,9 @@ def to_dataframe(self,start_time,
             zt_stag = getattr(self, height_var) # z(t)
             if agl:
                 zt_stag -= self.stationz
-            # both sets of vars include geopotential height, ph
-            varns_unstag = [varn for varn in varns if (not varn in staggered_vars)]
-            varns_stag = staggered_vars
+            varns_unstag = varns.copy()
```

Comment:
Here @phawbeck , I create separate lists of variables, to unstagger (`varns_unstag`) or leave alone (`varns_stag`). Again, this has become trivial upon realizing that w is already automatically destaggered.